boards/arm/lpc17xx_40xx/olimex-lpc1766stk/README.txt

```diff
@@ -135,11 +135,11 @@ Olimex LPC1766-STK development board
    LDS176 controller but it is 100% compatible with the Philips PCF8833).
    So how do you tell which controller you have? Some message boards have
    suggested that the LCD display be disassembled and the controller chip
-   measured with a digital caliper � well that�s getting a bit extreme.
+   measured with a digital caliper � well that�s getting a bit extreme.
 
-  "Here�s what I know. The Olimex boards have both display controllers
-   possible; if the LCD has a GE-12 sticker on it, it�s a Philips PCF8833.
-   If it has a GE-8 sticker, it�s an Epson controller. The older Sparkfun
+  "Here�s what I know. The Olimex boards have both display controllers
```

Comment:
Also here

Reply:
Yep you are right, I have a list, updating them now as well.

Reply:
helps if I save all my open files...  They are fixed but I have a lot of style violations now...